getting-started-finding-data/finding-data.md

```diff
@@ -7,20 +7,99 @@ url: https://data.norge.no/getting-started/finding-data
 
 # Finne data
 
+Data.norge.no gir en omfattende oversikt over datasett, begreper, API-er, informasjonsmodeller, tjenester og hendelser i offentlig sektor, inkludert data med juridiske begrensninger som taushetsplikt eller personopplysninger. Hensikten med denne oversikten er å gjøre det lettere for deg å finne, forstå og ta i bruk data fra offentlig sektor. For å finne frem blant alle beskrivelsene, tilbyr Data.norge.no forskjellige måter å søke på. Dersom du ikke finner dataen du leter etter, kan du registrere en etterspørsel.
+
 ## Søk
 
-### Kunstig intelligens-søk
+Felles datakatalog tilbyr tre forskjellige måter å søke etter data på: standard tekstsøk med filtrering, kunstig intelligens-søk og SPARQL-spørring.
+
+### Kunstig intelligens-søk (KI-søk)
+
+KI-søket bruker metadata fra data.norge.no og hjelper deg med å finne datasett. Ved å bruke naturlig språk kan du søke fritt uten å måtte kjenne til spesifikke datasettnavn, fagtermer eller tekniske formater.
+Prøv f.eks. "Hvilke områder i Norge er vernet?"
+
+[Du kan lese teknisk informasjon om KI-søket på data.norge.no her](https://github.com/Informasjonsforvaltning/fdk-llm-search-service).
 
-### Avansert søk
+### Standard søk med filter
+
+På [data.norge.no/search-all](data.norge.no/search-all) finner du et tekstsøk med muligheter for filtrering. Her kan du søke etter titler, beskrivelser og søkeord i databeskrivelsene. For å avgrense søket ditt, kan du bruke ulike filtre. Hvilke filtre som tilbys avhenger av ressurstype. Du velger ressurstype ved å bruke fanene under søkefeltet. Resultatet kan sorteres etter relevans eller publiseringsdato.
 
 ### SPARQL
 
+[SPARQL-tjenesten](https://data.norge.no/sparql) er det kraftigste og mest fleksible verktøyet for å søke i katalogen. Dette er et verktøy hvor du kan utføre egne spørringer i SPARQL på RDF-grafene til data.norge.no. Her kan du spørre etter datasett, begreper, API-er og informasjonsmodeller. Du kan også gjøre spørringer direkte til [SPARQL-endepunktet for data.norge.no](https://sparql.fellesdatakatalog.digdir.no/).
+
+#### Hva er SPARQL?
+
+[SPARQL](https://www.w3.org/TR/rdf-sparql-query/) er et spørrespråk som brukes til å hente data lagret i RDF-format. RDF Resource Description Framework) er en standardmodell for å beskrive data og ressurser på semantisk web. [Du kan lese mer om RDF her](lenke).
```

Comment:
Usikker på hva som blir best å peke til som ressurs for å lese mer om RDF, men kan vel starte med å peke til hovedsida for RDF hos w3.org og eventuelt endre til noe annet senere
```suggestion
[SPARQL](https://www.w3.org/TR/rdf-sparql-query/) er et spørrespråk som brukes til å hente data lagret i RDF-format. RDF Resource Description Framework) er en standardmodell for å beskrive data og ressurser på semantisk web. [Du kan lese mer om RDF her](https://www.w3.org/RDF).
```

Reply:
Enten som Nils Ove beskriver, eller om vi skal ha en egen side om RDF for å forklare det de trenger å vite på en enkel måte. F.eks. ta utgangspunkt i @terjesyl sin beskrivelse i ressursheftet til SIKT-konferansen.

Reply:
Jepp, tenker iallfall at det burde stå noe om det på den siden der vi forklarer hvordan komme i gang med høsting/skrive beskrivelser